### PR TITLE
docs: adjust the agent fee

### PR DESCRIPTION
**Pull request by istankovic**
Part of #1049.

### CODE REVIEW

File: docs/source/whitepaper.rst [hunk 1]
```diff
@@ -391,7 +391,7 @@ There are two fees that users need to pay to bridge their tokens:
 
 agent fee
     The fee paid in token being moved, covering the gas costs and rewarding
-    the liquidity provider. Variable and currently set to ``max(5e18, 0.1% of
+    the liquidity provider. Variable and currently set to ``max(1e18, 0.3% of
```
Comment (fredo) [inline docs/source/whitepaper.rst:394]: 1e18 is misleading in the context of USDC. But I don't have a better idea to represent the value, independently of the number of decimals. (i.e. for usdc it's 1e6)

Comment (fredo) [reply docs/source/whitepaper.rst:394]: maybe 1e[NUMBER_OF_DECIMALS]

Comment (istankovic) [reply docs/source/whitepaper.rst:394]: Yeah, I was wondering what to do here too...

Comment (istankovic) [reply docs/source/whitepaper.rst]: Expressed the min LP fee in terms of `token.decimals()`. Also slightly adjusted the presentation.

**Review by fredo — APPROVED** [review]

**Comment by istankovic** [comment]
@fredo should we also put this on `main`?

**Pull request merged**